resim/transforms/proto/fuzz_helpers.cc

```diff
@@ -12,4 +12,8 @@ bool verify_equality(const SE3 &a, const SE3 &b) {
   return unpack(a).is_approx(unpack(b));
 }
 
+bool verify_equality(const FramedVector_3 &a, const FramedVector_3 &b) {
+  return unpack(a).isApprox(unpack(b));
+}
```

Comment:
Do we need to check the frames as well? If not, I would recommend documenting it in the header.

Reply:
Another thought is that we should just put line 16 in the body of the test that needs it, since that's the test that needs the comparison without the frames. LMK your thoughts.

Reply:
I believe `isApprox` does check frames for `FramedVector_3`s!